controllers/storagecluster/cephblockpools.go

```diff
@@ -18,11 +18,22 @@ type ocsCephBlockPools struct{}
 // newCephBlockPoolInstances returns the cephBlockPool instances that should be created
 // on first run.
 func (r *StorageClusterReconciler) newCephBlockPoolInstances(initData *ocsv1.StorageCluster) ([]*cephv1.CephBlockPool, error) {
+	mirroringPeerSpec := &cephv1.MirroringPeerSpec{}
 	var mirroringSpec cephv1.MirroringSpec
+
 	if initData.Spec.Mirroring.Enabled {
 		mirroringSpec.Enabled = true
 		mirroringSpec.Mode = "image"
+
+		// ensures that peer cluster secret is exist and appending into cephBlockPool
```

Comment:
@GowthamShanmugam who will create this secret? 
If we are assuming the user will create the secret with the specified name this seems wrong If the two ceph clusters are created with the same name.

Reply:
The secret name which we are using here is different from the secret name that rook is using. remote peer cluster secret is stored in a local cluster with some secrete name which is understood only by OCS-operator and new token exchange agent.

Reply:
> If we are assuming the user will create the secret with the specified name this seems wrong 
>
We are not assuming it. We are enforcing it.
This seems to be the simplest solution but yes it has issues.

Do we want to take an alternative route (it'll probably involve adding more api spec or label selectors etc)?
@Madhu-1 @leseb WDYT?

Reply:
> If the two ceph clusters are created with the same name.
>
It'll be in different namespaces. We don't support multiple CephCluster yet in OCS so should we tackle that issue later?
I think that'll require more thought and design.

Reply:
If this is for a 1:1 cluster am fine but we need to change this when we support multiple clusters. Maybe we need to go for storage cluster spec change to add  more peers

Reply:
ack, Adding TODO on my name to handle multicluster case in future

Reply:
What does the secret contain and how many can be created? Will this be created by the UI when creating a storagecluster?